crates/swc_css_parser/src/parser/value/mod.rs

```diff
@@ -33,13 +33,27 @@ where
 
             let v = self.parse_one_value_inner()?;
 
+            println!("{:?}", self.input.cur());
+
             hi = v.span().hi;
             values.push(v);
             state = self.input.state();
 
             if !eat!(self, " ")
                 && !is_one_of!(
-                    self, ",", "function", "ident", "percent", "str", "#", "url", "[", "{", "("
+                    self,
```

Comment:
This will be refactored in near future, we should move recovery mode inside `Declaration` parsing, not here, so it will be simplified